src/python/dxpy/nextflow/nextflow_templates.py

```diff
@@ -39,10 +38,10 @@ def is_importer_job():
     return dxapp
 
 
-def get_nextflow_src(inputs=None, profile=None, resources_dir=None):
+def get_nextflow_src(custom_inputs=[], profile=None, resources_dir=None):
```

Comment:
How about setting custom_inputs to None and then in the code doing something like:
```
def my_func(working_list=None):
    if working_list is None: 
        working_list = []
```
See https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists